alembic/env.py

```diff
@@ -0,0 +1,80 @@
+from logging.config import fileConfig
+
+from sqlalchemy import engine_from_config
+from sqlalchemy import pool
+
+from alembic import context
+
+from database import Base
```

Comment:
The import statement 'from database import Base' assumes that 'Base' is defined in the 'database' module. Ensure that 'Base' is correctly imported from the module where it is defined, typically where SQLAlchemy models are declared.